pystac/item_collection.py

```diff
@@ -125,12 +125,8 @@ def __add__(self, other: object) -> "ItemCollection":
         if not isinstance(other, ItemCollection):
             return NotImplemented
 
-        combined = []
-        for item in self.items + other.items:
-            if item not in combined:
-                combined.append(item)
-
-        return ItemCollection(items=combined, clone_items=False)
+        combined = [*self.items, *other.items]
+        return ItemCollection(items=combined)
```

Comment:
```suggestion
        return ItemCollection(items=combined, clone_items=True)
```

Since we're calling our function, IMO it's best to be explicit about the behavior we want.

Reply:
Nope, I agree, `+` should mirror the default. Hopefully we don't have to change the default again but you never know. Thanks!